scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua

```diff
@@ -2,6 +2,12 @@
 -- Area: Jugner Forest
 --   NM: Meteormauler Zhagtegg
 -----------------------------------
+mixins = {require("scripts/mixins/job_special")}
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:setMod(dsp.mod.DMG, -50) -- "Has an innate damage reduction trait, about 50%"
```

Comment:
clopededia is wrong here. he had magic dmg redux only (as bg correctly shows)
https://www.bg-wiki.com/bg/Meteormauler_Zhagtegg

Reply:
Incidentally, the entire line of similar NMs all have either phys or mag dmg reduction and a posse (auto follows them) that does not always pop (separate timers maybe, who knows)

the quadav gets -50 physical

Reply:
(which I can't prove atm wiki's don't list, just that one anecdotal comment on every NM..from a blu)

Reply:
possibly their resistance is related to their posse being up, no way to know atm